GCMFormTableView.podspec

```diff
@@ -1,14 +1,14 @@
 Pod::Spec.new do |s|
   s.name         = "GCMFormTableView"
-  s.version      = "0.5.0"
+  s.version      = "0.5.1"
   s.summary      = "Form building framework."
   s.homepage     = "https://github.com/gamechanger/GCMFormTableView"
   s.author       = { "Jerry Hsu" => "jerry.hsu@gamechanger.io" }
-  s.source       = { :git => "https://github.com/gamechanger/GCMFormTableView.git", :tag => "0.5.0" }
+  s.source       = { :git => "https://github.com/gamechanger/GCMFormTableView.git", :tag => "0.5.1" }
   s.source_files = "GCMFormTableView/GCMFormTableView/*.{h,m}"
   s.license = { :type => 'MIT', :type => 'LICENSE' }
   s.requires_arc = true
   s.ios.deployment_target = '6.0'
   s.dependency 'GCMCocoaExtensions', '~> 0.2.0'
-  s.dependency 'ReactiveCocoa', '~> 2.3.1'
+  s.dependency 'ReactiveCocoa', '~> 2.4.7'
```

Comment:
So do I not commit changes to this file? Or do I just also need the change reflected in the `.erb` file?